src/test/java/tests/CartTests.java

```diff
@@ -7,11 +7,14 @@
 
     @Test
     public void testCartProductDelete() {
```

Comment:
cartProductDeleteTest

Reply:
name for classes - CartTest not (s).
in order to launch tests via maven properly.